apps/balancer-tools/src/app/poolsimulator/(components)/DepthCost.tsx

```diff
@@ -246,18 +246,22 @@ export function calculateDepthCost(
   if (!analysisToken) throw new Error("Analysis token not found");
   const tokenIn = poolSide === "in" ? analysisToken : pairToken;
   const tokenOut = poolSide === "in" ? pairToken : analysisToken;
-  const newSpotPrice = amm.spotPrice(tokenIn.symbol, tokenOut.symbol) * 1.02;
+  const currentSpotPrice = amm.spotPrice(tokenIn.symbol, tokenOut.symbol);
+  const newSpotPrice = currentSpotPrice * 1.02;
+  const spotPricePrecision = currentSpotPrice * 0.0002;
```

Comment:
just noticed that this is not being used. Is that correct?

Reply:
The spotPricePrecision? I forgot to add when merging the branches but now it is being used, right?

Reply:
yesss now I synced with main hahaha